style.css

```diff
@@ -0,0 +1,51 @@
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+  font-size: large;
+}
+
+#main {
+  background-color: #f5f5f5;
+  width: 100%;
+  text-align: center;
+  padding: 1.5rem;
+}
+
+#title {
+  font-size: 4rem;
+  margin-bottom: 0;
+}
+
+#img-div {
+  background: white;
+  padding: 10px;
+  margin: 0;
```

Comment:
There is no need to margin here as default value of margin is already zero.

Reply:
- [ ] Good job :smile: , very helpful comment .